src/pages/patientView/PatientViewGenePanelModal/PatientViewGenePanelModal.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { GenePanel } from 'shared/api/generated/CBioPortalAPI';
+import { observer } from 'mobx-react';
+import GenePanelModal from 'shared/components/GenePanelModal/GenePanelModal';
+import GenesList from './GenesList';
+import styles from './styles.module.scss';
+
+interface IPatientViewGenePanelModalProps {
+    genePanel: GenePanel;
+    show: boolean;
+    onHide: () => void;
+    columns?: number;
+}
+
+@observer
+export default class PatientViewGenePanelModal extends React.Component<
+    IPatientViewGenePanelModalProps,
+    {}
+> {
+    render() {
+        return (
+            <GenePanelModal
+                panelName="Gene Panel"
+                show={this.props.show}
+                onHide={this.props.onHide}
+                isLoading={!this.props.genePanel}
```

Comment:
the type of genePanel is `genePanel:GenePanel`, not `genePanel?:GenePanel` (i.e. `GenePanel|undefined`). so `!this.props.genePanel` is always `false`. If that's what you're going for, then you should just pass `isLoading={false}`

Reply:
That's true, my bad. I have removed the `isLoading` prop because it's optional.